playbook/workshops/remote_control/README.md

```diff
@@ -0,0 +1,615 @@
+
+Just click "Share" and then "Output only" to get the URL
+
+> ![](img/responsive.gif)
+
+Then type in the URL into your mobile browser. (You can also be clever and
+write a line of code to send the URL to your phone with a text message).
+
+### The final code
+
+Here's what [my code looks like]
+(http://jsbin.com/fawuda/120/edit?html,js,console,output)
+right now. In order for this example to work:
+
+- change the phone number to your phone number
+- set your twilio api key and token using
+  [the instructions here](#adding-your-twilio-sid-and-token)
+
+## Part II
+
+### Making the examples work
+
+**IMPORTANT:**
+
+All the examples also need to have a Twilio sid and auth token. However
+they use a different script than the above. To make the examples work for the
+rest of the tutorial follow these instructions:
+
+- Open [this JS Bin](http://jsbin.com/tetisa/1/edit?js,output).
+- Click "Run with JS"
+- When prompted, enter your Twilio sid and token
+
+> ![](img/entering_prompted_twilio_credentials.gif)
+
+### Actions
+
+#### Call And Play Music Through Phone
+
+```js
+Twilio.callAndPlay("555-555-5555", "http://a.tumblr.com/tumblr_lie8ewfdbO1qzbwpvo1.mp3");
+```
+
+[JS Bin](http://jsbin.com/fawuda/101/edit?js,console)
+
+```
+(don't forget to change the phone number and click the "run" button)
+```
+
+#### Play Music Through Your Computer
+
+```js
+var audio = new Audio("http://a.tumblr.com/tumblr_lie8ewfdbO1qzbwpvo1.mp3");
+audio.play();
+```
+
+[JS Bin](http://jsbin.com/fawuda/101/edit?js,console)
+
+```
+Random note: if you're trying to play a sound on mobile, it must be activated
+with a button press first before it can play.
+```
+
+### Triggers
+
+#### Magic Wand
+
+When the mobile phone is waved like a magic wand, have it trigger something.
+
+First include this javascript library at the bottom of the HTML page in an
+existing JS Bin:
+
+```html
+<script type="text/javascript" src="//bit.ly/wand_js"></script>
+```
+
+> ![](img/add_wand.gif)
+
+Then add this code in the JavaScript
+
+```js
+Wand.threshold = 5;
+Wand.onWave = function() {
+  alert("Wand Waved");
+  // INSERT CODE HERE YOU WANT TO RUN WHEN YOU HAVE THE WAND
+};
+```
+
+> ![](img/wand_setup.gif)
+
+Adjust
+
+```js
+Wand.threshold = 5;
+```
+
+The larger the number, the harder you have to fling the wand in order for it to
+trigger.
+
+For an iPhone 6, I personally like
+```js
+Wand.threshold = 20;
+```
+
+**Important Note**
+
+You will need to open the output view of JS Bin on your phone. After you do that
+try waving your phone around.
+
+- [Here is a bare bones working example](http://jsbin.com/fawuda/121/edit?html,js)
+- [This is the output URL](http://jsbin.com/fawuda/121)
+
+Here is a more complicated example
+
+- [link to the code](http://jsbin.com/fawuda/128/edit?html,js,output)
+- [link to the output that you should open on your phone]
+  (http://jsbin.com/fawuda/128)
+
+**Important Note** that for waving the wand to play a sound, you need to press
```

Comment:
Maybe in the wand library you could programmatically add a button, press it and then kill it? Or add a hidden button...